zbus_macros/src/lib.rs

```diff
@@ -267,7 +267,7 @@ pub fn proxy(attr: TokenStream, item: TokenStream) -> TokenStream {
 ///   important. You can use `out_args` to specify their names.
 ///
 /// * `proxy` - Use this to specify the [`macro@proxy`]-specific method sub-attributes (e.g
-///   `object`). The common sub-attributes (e.g `name`) are automatically forworded to the
+///   `object`). The common sub-attributes (e.g `name`) are automatically forwarded to the
```

Comment:
I appreciate your fixing the typo here but it's not relevant to this commit so if you could split it into a separate commit, that would be great (`git show --oneline && git reset HEAD^ && git add -p && git commit && git commit -a -C <commit-hash-from-git-show-command` should do it). :)